content/blog/deep-dive-data-fetcher-results.md

```diff
@@ -0,0 +1,177 @@
++++
+title = "graphql-java - In Depth - Part 1: DataFetcherResult"
+author = "Brad Baker"
+tags = []
+categories = []
+date = 2019-04-11T00:00:00+10:00
++++
+
+# graphql-java - In Depth series 
+
+Welcome to the new series "graphql-java - In Depth" where we will explore more specific graphql-java topics.  
```

Comment:
not sure we need to make a series. a stand alone blog about that topic might be enough.

Reply:
I try to write alway GraphQL Java instead of graphql-java